src/mappings/Rewards.ts

```diff
@@ -18,6 +19,8 @@ function extractArgsFromPayoutStakers(call: CallBase<AnyTuple>): [string, number
 }
 
 export async function handleReward(rewardEvent: SubstrateEvent): Promise<void> {
+    await handleRewardRestakeForAnalytics(rewardEvent)
+
     let element = await HistoryElement.get(eventId(rewardEvent))
```

Comment:
I think it would be logical to also move the other logic into a separate function like ```handleRewardForTxHistory```